configs/session.config.js

```diff
@@ -0,0 +1,10 @@
+const session = require('express-session')
+
+module.exports = app =>{
+    app.use(session({
+        secret: process.env.SESS_SECRET,
+        resave: false,
+        saveUninitialized: true,
+        cookie:{maxAge: 600000} // 600 * 1000 ms === 10 min
```

Comment:
Like that you actually did not only copy/past from the class example and personalized it 👍